hw/ipmi/ipmi-sel.c

```diff
@@ -528,6 +529,13 @@ static void sel_power(uint8_t power, void *context __unused)
 	}
 }
 
+static void sel_heartbeat(uint8_t heartbeat, void *context __unused)
+{
+	/* There is only one sub-command so no processing needed */
+	prlog(PR_NOTICE, "BMC issued heartbeat command: %02x\n",
```

Comment:
Can you drop this to PR_DEBUG? PR_NOTICE is printed to the console by default.

Reply:
Sure, let me do that quick